src/main/scala/com/qubole/spark/hiveacid/reader/hive/HiveAcidReader.scala

```diff
@@ -133,12 +190,10 @@ extends CastSupport with Reader with Logging {
 
   /**
    * @param hiveAcidMetadata - hive acid metadata of underlying table
-   * @param partitions - partitions for the table
    *
    * @return - Returns RDD on top of partitioned hive acid table
    */
-  def makeRDDForPartitionedTable(hiveAcidMetadata: HiveAcidMetadata,
```

Comment:
Any reason for removing partitions from method and making it member variable of class. I think one HiveAcidReader can be used multiple times to read 1, 2 or more partitions. HiveAcidReader should map to just tables and not to <Table, partitions>

Reply:
I think the logic is same. Earlier getTableReader used to create the HiveAcidReader and then call getRDD. Now i have just split it into two methods. The readerOptions is already created using the filters passed by user and from that the partitions list is generated. So i am not sure if we should change the partitions list after HiveAcidReader object is created.